dev/bitcoin/lnd-backup-test/build-image.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -e
+
+echo "=== LND Backup Image Setup for Testing ==="
+
+# Configuration
+ONLINE_IMAGE_REPO="us.gcr.io/galoy-org/lnd-backup"
+LOCAL_IMAGE_NAME="lnd-backup"
+IMAGE_TAG="latest"
+BUILD_LOCAL=${1:-false}
+
+if [ "$BUILD_LOCAL" = "true" ] || [ "$BUILD_LOCAL" = "--local" ]; then
+    echo "Building local backup image..."
+
+    echo "1. Building backup image..."
+    cd ../../../images/lnd-backup
```

Comment:
```suggestion
    pushd ../../../images/lnd-backup
```
pushd ... popd would be better here as the local build fails below in line 36:
```
./build-image.sh: line 36: cd: ../../../dev: No such file or directory
```